scripts/main.js

```diff
@@ -71,4 +72,68 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log("De afdeling sales heeft", departments.sales.numberOfEmployees, "medewerkers.");
+
+console.log("Marketing is een leuke afdeling om te werken.", departments.marketing.description)
+
+console.log("De afdeling Customer Service heeft", departments["customer-service"].numberOfEmployees, "medewerkers")
+
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager.", departments.sales.jobs[1].description)
+
+const userName = prompt('Hoi! Hoe heet je?');
+console.log(userName);
+//
+// document.getElementById('name').textContent = userName;
+
+const userInput = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+console.log(userInput);
+
+
```

Comment:
Ik heb perongeluk Sales in getikt. Dan krijg ik de error te zien. Het is mooi als je van mijn input kleine letters maakt met de methode toLowerCase().

userInput.toLowerCase()